### PR TITLE
Install Elasticsearch plugins

### LINKED ISSUE #1035: Localstack Elasticsearch plugin Ingest User Agent Processor not available
Plugin `Ingest User Agent Processor` is installed by default for Elasticsearch (ELK) on AWS. It is not the case in Localstack and think we basically expect it.

In addition, I was not able to install it manually through command `bin/elasticsearch-plugin install ingest-user-agent` as bin/elasticsearch-plugin is missing.

### PR DESCRIPTION
**Pull request by dmnlk**
Fix #1035

install ES plugin by  supports Amazon Elasticsearch Service
https://docs.aws.amazon.com/elasticsearch-service/latest/developerguide/aes-supported-plugins.html

### CODE REVIEW

File: localstack/services/install.py [hunk 1]
```diff
@@ -47,6 +47,13 @@ def install_elasticsearch():
             mkdir(dir_path)
             chmod_r(dir_path, 0o777)
 
+        # install default plugins
+        for plugin in ELASTICSEARCH_PLUGIN_LIST:
+            if is_alpine():
+                # https://github.com/pires/docker-elasticsearch/issues/56
+                os.environ['ES_TMPDIR '] = '/tmp'
```
Comment (whummer) [inline localstack/services/install.py]: nit: better remove the additional space in `os.environ['ES_TMPDIR']`

File: localstack/services/install.py [hunk 2]
```diff
@@ -47,6 +47,13 @@ def install_elasticsearch():
...
+            run('%s/bin/elasticsearch-plugin install %s' % (INSTALL_DIR_ES, plugin))
```
Comment (whummer) [inline localstack/services/install.py]: For cross-platform compatibility, perhaps better:
```
run('%s install %s' % (os.path.join('bin', 'elasticsearch-plugin', INSTALL_DIR_ES), plugin))
```

Comment (dmnlk) [reply localstack/services/install.py]: fixed!

**Comment by whummer** [comment]
Thanks for this PR @dmnlk ! Can you please show a quick size comparison of the old vs new Docker images (base image and final image). Would like to get a rough understanding how much additional space these plugins take. Thanks

**Comment by dmnlk** [comment]
Old Version
```
[git][* master]:~/src/github.com/localstack/localstack/ git show HEAD                                                                                                                                                              [18:40:10]
commit 1c5f2e9650155a839cc842a9cd07faf3e76ed5d2 (HEAD -> master, origin/master, origin/HEAD)
Author: Olle Jonsson <olle.jonsson@gmail.com>
Date:   Sun Jan 6 22:35:43 2019 +0100

    README: Use SVG build badge

diff --git a/README.md b/README.md
index 8ed7643..f7927ba 100644
--- a/README.md
+++ b/README.md
@@ -1,4 +1,4 @@
-[![Build Status](https://travis-ci.org/localstack/localstack.png)](https://travis-ci.org/localstack/localstack) [![Backers on Open Collective](https://opencollective.com/localstack/backers/badge.svg)](#backers) [![Sponsors on Open Collective](https://opencollective.com/localstack/sponsors/badge.svg)](#sponsors) [![Coverage Status](https://coveralls.io/repos/github/localstack/localstack/badge.svg?branch=master)](https://coveralls.io/github/localstack/localstack?branch=master)
+[![Build Status](https://travis-ci.org/localstack/localstack.svg)](https://travis-ci.org/localstack/localstack) [![Backers on Open Collective](https://opencollective.com/localstack/backers/badge.svg)](#backers) [![Sponsors on Open Collective](https://opencollective.com/localstack/sponsors/badge.svg)](#sponsors) [![Coverage Status](https://coveralls.io/repos/github/localstack/localstack/badge.svg?branch=master)](https://coveralls.io/github/localstack/localstack?branch=master)
 [![Gitter](https://img.shields.io/gitter/room/localstack/Platform.svg)](https://gitter.im/localstack/Platform)
 [![PyPI Version](https://badge.fury.io/py/localstack.svg)](https://badge.fury.io/py/localstack)
 [![PyPI License](https://img.shields.io/pypi/l/localstack.svg)](https://img.shields.io/pypi/l/localstack.svg)
```

only changed Java Version(can't build)
```
[git][* master]:~/src/github.com/localstack/localstack/ git diff                                                                                                                                                                   [18:47:38]
diff --git a/bin/Dockerfile.base b/bin/Dockerfile.base
index 1a605a3..3956d02 100644
--- a/bin/Dockerfile.base
+++ b/bin/Dockerfile.base
@@ -26,8 +26,8 @@ RUN { \
     && chmod +x /usr/local/bin/docker-java-home
 ENV JAVA_HOME /usr/lib/jvm/java-1.8-openjdk
 ENV PATH $PATH:/usr/lib/jvm/java-1.8-openjdk/jre/bin:/usr/lib/jvm/java-1.8-openjdk/bin
-ENV JAVA_VERSION 8u171
-ENV JAVA_ALPINE_VERSION 8.181.13-r0
+ENV JAVA_VERSION 8u191
+ENV JAVA_ALPINE_VERSION 8.191.12-r0
 RUN set -x && apk add --no-cache openjdk8="$JAVA_ALPINE_VERSION" \
     && [ "$JAVA_HOME" = "$(docker-java-home)" ]
```

base image?
```
[git][* master]:~/src/github.com/localstack/localstack/ docker images                                                                                                                                                              [18:47:51]
REPOSITORY                                                                  TAG                 IMAGE ID            CREATED             SIZE
localstack/java-maven-node-python                                           0.8.10              bc2ca0f2db84        43 seconds ago      507MB
localstack/java-maven-node-python                                           latest              bc2ca0f2db84        43 seconds ago      507MB
<none>                                                                      <none>              33a5f973e96f        50 seconds ago      751MB
<none>                                                                      <none>              788fd6148c24        6 minutes ago       387MB
```

final image


```
[git][* master]:~/src/github.com/localstack/localstack/ docker images                                                                                                                                                              [18:55:01]
REPOSITORY                                                                  TAG                 IMAGE ID            CREATED             SIZE
<none>                                                                      <none>              321d3f107722        58 seconds ago      1.04GB
localstack/localstack                                                       0.8.10              1b56b3258fe4        4 minutes ago       1e+03MB
localstack/localstack                                                       latest              1b56b3258fe4        4 minutes ago       1e+03MB
```

**Comment by dmnlk** [comment]
I compared ElasticSearch dir size
old ver(not installed ES plugins

```
bash-4.4# du -h -d 1 localstack/infra/elasticsearch/
20.0K	localstack/infra/elasticsearch/config
8.6M	localstack/infra/elasticsearch/modules
22.2M	localstack/infra/elasticsearch/lib
260.0K	localstack/infra/elasticsearch/bin
4.0K	localstack/infra/elasticsearch/logs
4.0K	localstack/infra/elasticsearch/plugins
31.3M	localstack/infra/elasticsearch/
```

new ver(installeed ES Plugins)


```
bash-4.4# du -h -d 1 localstack/infra/elasticsearch/
20.0K	localstack/infra/elasticsearch/config
8.6M	localstack/infra/elasticsearch/modules
22.2M	localstack/infra/elasticsearch/lib
260.0K	localstack/infra/elasticsearch/bin
4.0K	localstack/infra/elasticsearch/logs
50.3M	localstack/infra/elasticsearch/plugins
81.6M	localstack/infra/elasticsearch/
```

increase 50MB

**Comment by whummer** [comment]
Ok great, thanks. I'd say 50MB increase should be acceptable given that this change significantly improves compatibility with AWS. Thanks again!

**Comment by dmnlk** [comment]
thanks!

**Pull request merged**